tests/test_linkml_files.py

```diff
@@ -1,64 +1,196 @@
+           GITHUB_IO_PATH_FOR(Source.TYPES, Format.NATIVE_JSONLD)
+    assert "https://raw.githubusercontent.com/linkml/linkml-model/main/linkml_model/jsonld/meta.context.jsonld" == \
+           GITHUB_PATH_FOR(Source.META, Format.NATIVE_JSONLD, ReleaseTag.LATEST)
+    assert "https://raw.githubusercontent.com/linkml/linkml-model/testing_branch/linkml_model/owl/mappings.owl.ttl" == \
+           GITHUB_PATH_FOR(Source.MAPPINGS, Format.OWL, branch="testing_branch")
+
+
+SKIP_GITHUB_API = False
+
+
+@pytest.mark.skipif(SKIP_GITHUB_API, reason="Github API tests skipped")
+def test_github_specific_rules():
+    """
+        Test accesses that require GitHub API to access
         This is separate because we can only do so many tests per hour w/o getting a 403
         """
-        self.assertEqual("https://raw.githubusercontent.com/linkml/linkml-model/f30637f5a585f3fc4b12fd3dbb3e7e95108d4b42/jsonld/meta.model.context.jsonld",
-                         GITHUB_PATH_FOR(Source.META, Format.NATIVE_JSONLD, "v0.0.1"))
-        current_loc = re.sub(r'linkml-model/[0-9a-f]*/', 'linkml-model/SHA/', GITHUB_PATH_FOR(Source.TYPES, Format.YAML))
-        self.assertEqual("https://raw.githubusercontent.com/linkml/linkml-model/SHA/model/schema/types.yaml", current_loc)
-        # TODO: We may want to raise an error here?
-        self.assertEqual('https://raw.githubusercontent.com/linkml/linkml-model/missing_branch/owl/mappings.owl.ttl',
-                         GITHUB_PATH_FOR(Source.MAPPINGS, Format.OWL, branch="missing_branch"))
-
-        with self.assertRaises(ValueError) as e:
-            GITHUB_PATH_FOR(Source.META, Format.RDF, "vv0.0.1")
-        self.assertEqual("Tag: vv0.0.1 not found!", str(e.exception))
-
-    def test_shorthand_paths(self):
-        self.assertEqual('meta', str(fileloc.meta))
-        self.assertEqual('meta.yaml', str(fileloc.meta.yaml))
-        self.assertEqual('meta.py', str(fileloc.meta.python))
-        self.assertEqual(abspath('linkml_model/model/schema/meta.yaml'), str(fileloc.meta.yaml.file))
-        self.assertEqual('https://linkml.github.io/linkml-model/model/schema/meta.yaml', str(fileloc.meta.yaml.github_loc()))
-        self.assertEqual('https://raw.githubusercontent.com/linkml/linkml-model/f30637f5a585f3fc4b12fd3dbb3e7e95108d4b42/model/schema/meta.yaml', str(fileloc.meta.yaml.github_loc('v0.0.1')))
-
-
-if __name__ == '__main__':
-    unittest.main()
+    assert "https://raw.githubusercontent.com/linkml/linkml-model/f30637f5a585f3fc4b12fd3dbb3e7e95108d4b42/jsonld/meta.context.jsonld" == \
+        GITHUB_PATH_FOR(Source.META, Format.NATIVE_JSONLD, "v0.0.1")
+    current_loc = re.sub(r'linkml-model/[0-9a-f]*/', 'linkml-model/SHA/', GITHUB_PATH_FOR(Source.TYPES, Format.YAML))
+    assert "https://raw.githubusercontent.com/linkml/linkml-model/SHA/model/schema/types.yaml" == current_loc
+    assert 'https://raw.githubusercontent.com/linkml/linkml-model/missing_branch/linkml_model/owl/mappings.owl.ttl' == \
```

Comment:
note: change in path here

Reply:
the thing more worthy of fixing to me here is `/SHA/model` and `missing_branch/linkml_model` having different directory structure, but if the URLs resolve and they give us the thing we were after then they are the correct URLs in my book!